src/main/scala/uk/gov/nationalarchives/DynamoFormatters.scala

```diff
@@ -182,4 +188,10 @@ object DynamoFormatters {
 
   case object File extends Type
 
+  sealed trait RepresentationType
```

Comment:
I would like to because otherwise it's a pain seperating the dependencies with the same name like in the reconciler. The problem is that there's no dependency on the Preservica client in the dynamo formatters and it doesn't feel like there should be either. 
I think the best we can do is rename these so they don't clash with the Preservica ones. `FileRepresentationType` or something like that. What do you think?

Reply:
Hmm, but they are both representation types of files. is there an adjective that could differentiate the two e.g. "{X}RepresentationType"? If not, then we can just use `FileRepresentationType`.

Reply:
They are both representation types of files so I think `FileRepresentationType` is ok.

Reply:
I'll change the case objects I think to `RepresentationTypePreservation` and `RepresentationTypeAccess` to avoid clashes as well I think.

Reply:
Maybe the other way around? like `PreservationRepresentationType` and `AccessRepresentationType`?